docs/website/docs/developers/general/contributing.md

```diff
@@ -243,12 +243,24 @@ Write | **Established project contributors should request this access**<br>:mate
 Maintain/Admin | :material-check: Can [edit repository settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features)<br>:material-check: Can push to [protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) | Added case-by-case
 
 All access tiers first require joining the
-[iree-org GitHub organization](https://github.com/iree-org/).
+[iree-org GitHub organization](https://github.com/iree-org/). To request
+membership in iree-org, send an email to
+<iree-github-requests@lists.lfaidata.foundation> with this template:
```

Comment:
I've started this mailing list with just myself and @marbre for testing purposes. Others can either add themselves to the list via the LFX project center or I can add them myself.

Looking mostly at @benvanik and @stellaraccident , but other organization owners are welcome too.

We don't need the full set of owners monitoring the list, but I do want at least a few people so we don't have a single point of failure.